index.html

```diff
@@ -8,6 +8,53 @@
     <title>Rocket Academy</title>
   </head>
   <body>
-    <h1>Hello, world!</h1>
+    <h1 id="header">Welcome to Yoman's portfolio</h1>
+    <h2 id="sub-header">Mini projects done in 2022 </h2>
+    <br>
+
+    <div id="projects-container" class="container">
```

Comment:
Nice use of id and class combination here. Maybe we could rename the container class a bit more, to signify what the class really is for. Maybe something like "flex--row-center" or the like. Following BEM naming convention can be helpful. 

http://getbem.com/introduction/